code-challenges.js

```diff
@@ -2,8 +2,14 @@
 
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
+const fibo = (num) => {
+  let newArr = [1,1]
+  for(let i=1; i<num;i++){
+      newArr.push(newArr[i]+newArr[i-1])
+  }
+  return newArr
+}
+console.log(fibo(9))
```

Comment:
woo! maybe think about adding some comments to this code though since it's not immediately obvious what's happening